kustomize/envs/Development/secrets.properties

```diff
@@ -0,0 +1,2 @@
+database-password=#{database-password}

```

Comment:
Is there any reason these variables couldnt follow the `.` notation used for other variable substitution? 

e.g. `#{Project.Database.Password}`